gateway/src/apicast/policy_loader.lua

```diff
@@ -0,0 +1,199 @@
+--- Policy loader
+-- This module loads a policy defined by its name and version.
+-- It uses sandboxed require to isolate dependencies and not mutate global state.
```

Comment:
Maybe emphasize somehow that this loader allows us to load several versions of the same policy without problems?

Reply:
Yep. It even allows loading the same policy several times without global state. Will emphasize that.